pootle/apps/pootle_app/management/commands/update_stores.py

```diff
@@ -45,9 +45,18 @@ def handle_translation_project(self, translation_project, **options):
 
     def handle_store(self, store, **options):
         overwrite = options.get('overwrite', False)
-        force = options.get('force', False)
-
-        store.update(overwrite=overwrite, only_newer=not force)
+        disk_mtime = store.get_file_mtime()
+        if disk_mtime is None:
+            logging.debug(u"File missing from disk, skipping "
+                          u"%s" % store.pootle_path)
+            return
+        if not options.get('force', False):
```

Comment:
Note as long as ``--force`` has a default value this is always defined.  So no need to use ``.get`` with [default values](https://docs.python.org/2/library/optparse.html#default-values) outside of the actual option definition.  And it should be ``options.force`` when you actually need to use the setting.